CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.18)
+cmake_minimum_required(VERSION 3.22) # FIXME: 3.24.2 / 3.25
```

Comment:
You probably checked already, but > 3.22 would exclude Ubuntu 22.04 (unless they backport), which is still supported for a while...

Reply:
> we _could_ just drop support for `Ubuntu 22.04` right away

Don't think that'll be a realistic option for the summer cycle... IIRC the last time around it took several months after the release for GitHub runners to become available...

Reply:
I don't quite follow, why on earth would it ever matter what the _default_ is on github runners?
The only thing the github linux runners should be used for, is to run docker on them,
and from there on you are free to pick your baseline image.
See https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container
That's what i'd always done in this repo. Is that not what happens for darktable? o_O

Reply:
Note that I said _availability_, without mentioning any default.

Reply:
> And no, not in a container, dt uses straight runners.

Well, "you" might want to get that fixed then :)